filecoin-hashers/Cargo.toml

```diff
@@ -32,7 +34,7 @@ opencl = ["bellperson/opencl", "neptune/opencl"]
 
 # available hashers
 blake2s = ["blake2s_simd"]
-poseidon = ["neptune", "lazy_static"]
+poseidon = ["neptune", "lazy_static", "typemap"]
```

Comment:
Note: `typemap` allows us to lookup the Poseidon constants for a field and arity.